tests/Agents/Conversation/ConversationBase.php

```diff
@@ -118,6 +123,57 @@ public function testSend(): void
         $this->assertNotEmpty($conversationMessages[1]->getContent());
     }
 
+    public function testSchema(): void
+    {
+        if (
+            $this->adapter->getModel() === GEMINI::MODEL_GEMINI_2_0_FLASH_LITE ||
```

Comment:
Never check for specific model names, instead implement an abstract method or interface that forces each class to have a method like `isSchemaSupported():bool`